src/app/components/common/feature/feature.component.html

```diff
@@ -54,6 +55,7 @@ <h3>Who we are</h3>
                 <div class="feature-content">
                   <h3>What we do</h3>
                   <p>Oraganizing amazing tech events, Speaking and Mentoring</p>
+                  <a href="#"><button id="exporeBtn" class="btn btn-common nav-link">More <i class="lni lni-chevron-right"></i></button></a>
```

Comment:
No need more button here. remove this and its styles. i will merge it with the new social links only